common/tools/patch_tool_test.sh

```diff
@@ -13,6 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# --- begin runfiles.bash initialization v2 ---
+# Copy-pasted from the Bazel Bash runfiles library v2.
```

Comment:
Since we need this snippet in all tests can we put that in one place (maybe `bazel:runfile_support`) somewhere and then just source in all the tests where we need it ?

Reply:
The problem is how to find that snippet on Windows? The answer is `rlocation`, which the snippet import... We could implement our own version, but that is kinda what the snippet does already. The snippet is the way [Bazel recommends](https://github.com/bazelbuild/bazel/blob/9ad5aeddb24b8c6ca45ec3fbe07b00d80cd8fb59/tools/bash/runfiles/runfiles.bash) it AFAIK.

There is probably a way using a genrule that would wrap the script to run around some initialization. I will give it a try.

Reply:
ha, yes, it wouldn't be found, good point.
But if we find something to reduce the repetetive boilderplate would probably good.

Reply:
I added a convenience `sh_test` wrapper that loads the bash runfiles library before calling the test. That reduces boilerplate code significantly.